mvg/mvg.py

```diff
@@ -396,7 +396,7 @@ def create_measurement(
 
     # in example
     def read_measurements(self, sid: str):
-        """Retrieves all measurements for all timestamps for a source.
+        """Retrieves all measurements (all timestamps) for a source.
```

Comment:
Would not be a good idea just to add in the function description "...(all timestamps and metadata)..."

Reply:
@SergioMDCB Good point. I assume you mean "Would be a good idea ..."